### ISSUE TITLE
Support collectors on a `command` in a `TestSuite`

### ISSUE THREAD

**Issue by porridge** [kudobuilder/kuttl#190]
Not sure if I got this right (the docs seem to be a little [stale](kuttl.dev#20)), but it sounds to me like `collectors` can currently only be specified on a `TestAssert` which in turn can only be specified on the `NN-assert.yaml` level.

It would be great if they could also be specified on a `command` specified in a `kuttl-test.yaml`.

Otherwise one needs to do ugly things like:
- wrap a command in an extra shell script which does `if ! main-command; then dump some debug info;fi`, or
- artificially set `ignoreFailure: true` on the command and then add another subsequent command which checks state and dumps debug info before failing.

_Originally posted by @porridge in https://github.com/kudobuilder/kuttl/issues/174#issuecomment-660968971_

**Comment by kensipe** [comment]
yeah... I meant to re-open the original post after the updated collectors landed... in order to capture this... but capturing the addition details here seems better!  thanks for following it up!
